devcycle-ruby-server-sdk.gemspec

```diff
@@ -21,12 +21,12 @@ Gem::Specification.new do |s|
   s.summary     = "DevCycle Bucketing API Ruby Gem"
   s.description = "DevCycle Ruby Server SDK, for interacting with feature flags created with the DevCycle platform."
   s.license     = "MIT"
-  s.required_ruby_version = ">= 2.4"
+  s.required_ruby_version = ">= 3.1"
```

Comment:
So are we going to need to mark this as a major version if we are going from 2.4 -> 3.1?

Reply:
We can - but 3.0 is EOL https://endoflife.date/ruby
https://www.ruby-lang.org/en/downloads/

Reply:
yea like if we make this big jump in supported versions we probably need to make it a major version.

Reply:
kk - i'll mark major when we release